src/Components/Restaurante/ConfirmacaoPedido/ConfirmacaoPedido.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
```

Comment:
Não entendi esse arquivo aqui... Não encontrei referência a ele em nenhum outro lugar. Nesse caso, se ele realmente não é usado, não é interessante deixar código "morto" aqui. Se for ser usado no futuro ou é importante que seja mantido por qualquer motivo, indique com um comentário no topo do arquivo.